packages/protocol/contracts/common/interfaces/IAccounts.sol

```diff
@@ -0,0 +1,20 @@
+pragma solidity ^0.5.3;
+
+
+interface IAccounts {
+  function isAccount(address) external view returns (bool);
+  function getAccountFromVoteSigner(address) external view returns (address);
```

Comment:
how do you feel about "voteSignerToAccount()" naming style?

Reply:
I feel ok with any naming, happy to go with whatever y'all think is best @asaj

Reply:
I'm easy